inc/admin/customizer.php

```diff
@@ -119,9 +121,10 @@ function alcatraz_customize_register( $wp_customize ) {
 	$wp_customize->add_setting(
 		'alcatraz_options[page_banner_widget_area]',
 		array(
-			'default'    => 0,
-			'type'       => 'option',
-			'capability' => 'edit_theme_options',
+			'default'           => 0,
+			'type'              => 'option',
+			'capability'        => 'edit_theme_options',
+			'sanitize_callback' => 'is_int',
```

Comment:
This should be `intval` I think. `is_int` may be working because it will return 1 or 0 as true/false, but the actual sanitization we want to perform is simply to force it to be an integer.